server.js

```diff
@@ -70,19 +64,19 @@ app.prepare().then(() => {
       res.end()
     } else if (/^\/(deb|rpm)\/.*/i.test(pathname)) {
       // path /(deb|rpm)/... -> corresponding S3 bucket
-      res.writeHead(301, {
+      res.writeHead(303, {
         Location:
           'https://s3-us-east-2.amazonaws.com/dvc-s3-repo/' +
           pathname.substring(1)
       })
       res.end()
     } else if (/^\/(help|chat)\/?$/i.test(pathname)) {
       // path /(help|chat) -> Discord chat invite
-      res.writeHead(301, { Location: 'https://discordapp.com/invite/dvwXA2N' })
+      res.writeHead(303, { Location: 'https://discordapp.com/invite/dvwXA2N' })
       res.end()
     } else if (/^\/(docs|documentation)(\/.*)?$/i.test(pathname)) {
       // path /docs... or /documentation... -> /doc...
-      res.writeHead(301, {
+      res.writeHead(303, {
```

Comment:
should be 301 here?

Reply:
Hmmm probably not? docs.and documentation. subdomains are "structural" redirects, not previously existing pages we would need to transfer "link juice" (SEO stuff) over to a new address. I think this is the criteria we came up with to decide which response status code to use. I may be confused so up to you, please lmk.

Reply:
but they are not subdomains in this case, right?

Reply:
Ah yeah sorry I meant /docs and /documentation **paths**.

Reply:
So, the are not subdomains. There are external links out there on other websites like dataversioncontrol.com/documentation/tutorial and we should be notifying Google explicitly that we moved it to the new domain/path.

Reply:
> There are external links out there on other websites like dataversioncontrol.com/documentation/tutorial

So this would be redirected to dvc.org/documentation/tutorial ? I wasn't aware. I guess in that case `301` is more appropriate. Changed in cc376793.